app/views/devise/registrations/new.html.erb

```diff
@@ -4,9 +4,9 @@
   <%= devise_error_messages! %>
 
   <div class="field">
-      <%= f.label :username %>
-      <%= f.text_field :username, autofocus: true %>
-    </div>
+    <%= f.label :username %>
+    <%= f.text_field :username, autofocus: true %>
+  </div>
 
   <div class="field">
     <%= f.label :email %>
```

Comment:
It looks like this deleted and added the same lines? I think I'm missing something.